src/bio/bio_xstream.c

```diff
@@ -1162,7 +1166,7 @@ find_xs_bio_dev(struct bio_xs_context *ctxt, int tgt_id, enum smd_dev_type st,
 	 * Lookup device mapped to @tgt_id and @st in the per-server
 	 * metadata, if found, create blobstore on the mapped device.
 	 */
-	if (tgt_id != SYS_TGT_ID) {
+	if (tgt_id != DSS_SYS_TGT_ID) {
```

Comment:
This should be defined in SMD or BIO, and standalone VOS needs be considered too. We can leave standalone VOS support in follow-in PR though.

Reply:
OK!